test/targets/create-target.e2e-spec.ts

```diff
@@ -77,8 +78,21 @@ describe('POST /targets', () => {
       })
 
       it('should return the newly created target', async () => {
-        const target = await targets.last()
-        expect(response.body).toEqual(TargetDto.from(target))
+        firstTarget = await targets.last()
+        expect(response.body.target).toEqual(TargetDto.from(firstTarget))
+      })
+      
+      describe('when there is another target nearby with the same topic', () => {
+        it('should create a new match', async () => {
+          response = await request(app.getHttpServer())
+            .post('/targets')
+            .set('Authorization', `Bearer ${accessToken}`)
+            .send({ ...mockTarget2, topicId: mockTopic.id })
+            .expect('Content-Type', /json/)
+            .expect(201)
+          firstTarget.topic = undefined // Matches in the response don't include a topic
```

Comment:
This would be clearer by creating another target called `expectedMatch` or something like that, or just checking that the ids are equal which is what you really want to validate